subsys/fcb/fcb.c

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
Dont think this makes a subsystem on its own, this belongs rather into something else, maybe subsys/storage.

Reply:
or under filesystem... (fs)

Reply:
I suggest we place it under filesystem